tools/src/dwarfs_main.cpp

```diff
@@ -1336,6 +1342,15 @@ int option_hdl(void* data, char const* arg, int key,
   return 1;
 }
 
+// automount ok if dir doesn't exist or it's empty
+// leaving permission and more serious checks to fuse
+inline bool is_good_mount_path(std::filesystem::path& path) {
+  if (!std::filesystem::exists(path) ||
+    (std::filesystem::is_directory(path) && std::filesystem::is_empty(path)))
+    return true;
+  return false;
```

Comment:
```suggestion
  return !std::filesystem::exists(path) ||
    (std::filesystem::is_directory(path) && std::filesystem::is_empty(path));
```

Although that still leaves some questions:

- The `is_empty()` check isn't strictly required, you can always mount over a non-empty dir. But I don't mind the extra paranoia.
- But, more importantly, if it does *not* exist, the parent should be a directory. I guess that's kinda implicitly given by the rest of the code, but explicit is better than implicit. :)